prompt_guardian/server.py

```diff
@@ -202,14 +208,32 @@ async def check_prompt(prompt_check_request: CheckPromptRequest, request: Reques
     prompt = prompt_check_request.text
     url_manager = request.app.state.url_manager
 
-    if url_manager.enabled:
-        url_status = check_url_status(prompt, url_manager)
-    else:
-        url_status = "URL checking is disabled"
-    openai_prompt_status = await check_prompt_status(prompt, prompt_guardian_app.state.openai)
-    gemini_prompt_status = await check_prompt_status(prompt, prompt_guardian_app.state.gemini)
-    azure_prompt_status = await check_prompt_status(prompt, prompt_guardian_app.state.azure)
-    threats = check_threats(prompt, request.app.state.class_instance)
+    # default statuses - these will be returned to the client when the functionalities are not selected in the request
+    url_status = "Prompt check for malware URLs is disabled per client request"
+    openai_prompt_status = "Prompt injection detection with OpenAI is disabled per client request"
+    gemini_prompt_status = "Prompt injection detection with Gemini is disabled per client request"
+    azure_prompt_status = "Prompt injection detection with Azure is disabled per client request"
+    threats = "Prompt check for DLP wit Umbrella is disabled per client request"
```

Comment:
> wit

looks like we missed a typo, should be `with`, can be fixed next time

Reply:
https://github.com/BenderScript/PromptGuardian/pull/